src/Microsoft.TestPlatform.CrossPlatEngine/Microsoft.TestPlatform.CrossPlatEngine.csproj

```diff
@@ -25,6 +25,7 @@
     <ProjectReference Include="..\Microsoft.TestPlatform.CoreUtilities\Microsoft.TestPlatform.CoreUtilities.csproj">
       <FromP2P>true</FromP2P>
     </ProjectReference>
+    <PackageReference Include="System.Runtime.CompilerServices.Unsafe" Version="4.5.2" />
```

Comment:
Explain

Reply:
Removed from here but needed in TestHostProvider.csproj